crates/orchestrator/src/generators/bootnode_addr.rs

```diff
@@ -83,7 +87,13 @@ mod tests {
             .iter()
             .map(|x| x.to_string())
             .collect();
-        let bootnode_addr = generate(peer_id, &LOCALHOST, 5678, args.iter().as_ref(), &None);
+        let bootnode_addr = generate(
+            peer_id,
+            &LOCALHOST.to_string(),
+            5678,
```

Comment:
Maybe in the future we could create something like ```type Host = (IpAddr::V4, u16)```

Reply:
I think keep addr/port as two args is ok, but we can re-visit in the future.